src/yadmark/benchmark/external/apfel_utils.py

```diff
@@ -25,10 +24,18 @@ def load_apfel(theory, observables, pdf="ToyLH", use_external_grid=False):
         theory, observables, pdf, use_external_grid=use_external_grid
     )
 
+    is_polarized = ["g" in obs_name for obs_name in observables["observables"]]
```

Comment:
need to be fixed, see [here](https://github.com/NNPDF/yadism/pull/180#discussion_r1135285051)

Reply:
`obs_name.startswith("g")`

Reply:
Yes, this will be fixed by @RoyStegeman in #180 after which we'll merge the changes here.